query-node/mappings/src/content/channel.ts

```diff
@@ -332,3 +345,54 @@ export async function content_ChannelAgentRemarked(ctx: EventContext & StoreCont
     await updateMetaprotocolTransactionStatus(store, metaprotocolTxIdentifier, statusErrored, e)
   }
 }
+
+export async function ChannelTransferAccepted({ store, event }: EventContext & StoreContext): Promise<void> {
```

Comment:
Extrinsics related to channel transfers are disabled right now, so don't think this mapping is needed.

Reply:
Are you sure about this? I haven't found any place in runtime code that suggests that in `carthage` branch.

Anyway, I've removed the event. It can be reintroduced easily by reverting https://github.com/Joystream/joystream/pull/4150/commits/f0fbb1c01426555ca0446228f1ae62aa0b0402ec.

Reply:
Yes, some extrinsics were disabled in #3914 including channel transfers actions